src/ansys/dpf/core/operators/math/modal_participation.py

```diff
@@ -43,7 +43,7 @@ class modal_participation(Operator):
     >>> op.inputs.mode_shapes.connect(my_mode_shapes)
     >>> my_ponderation = dpf.Field()
     >>> op.inputs.ponderation.connect(my_ponderation)
-    >>> my_force_label_space = dpf.LabelSpace()
+    >>> my_force_label_space = dpf.dict()
```

Comment:
@jose-henriquezroa @PProfizi the mustache maybe needs to be updated to reflect that this is dict() and not dpf.dict, right?

Reply:
@rafacanton indeed, this is will look weird otherwise

Reply:
@PProfizi @jose-henriquezroa OK, I'll close the PR until that change is done